Assets/Scripts/Room/Chat/ChatMessage.cs

```diff
@@ -0,0 +1,122 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Localization.Settings;
+using System.Collections;
+using Assets.Scripts.UI.Input;
+
+namespace Assets.Scripts.Room.Chat
+{
+    public class ChatMessage : MonoBehaviour
```

Comment:
No need to do a new line after every bracket

Reply:
But it looks pretty.